shell/edit/fleet.cattle.io.gitrepo.vue

```diff
@@ -512,6 +546,40 @@ export default {
       }
     },
 
+    enablePolling(value) {
+      if (value) {
+        delete this.value.spec.disablePolling;
+      } else {
+        this.value.spec.disablePolling = true;
+      }
+    },
+
+    updatePollingInterval(value) {
+      let interval = (value || DEFAULT_POLLING_INTERVAL).toString();
```

Comment:
something odd happens here with `interval`. it's only used in the final else block, but also set in the preceding if block (but ignored). logic needs re-working